### DISCUSSION TITLE
edge浏览器pdf翻译失效

### DISCUSSION THREAD

**Discussion by Zonefat** [chunibyocola/sc-translator-crx#67] [Q&A] [answered]
设置中已启用 PDF 阅读器，使用edge自带的pdf阅读器无法实现翻译功能。
例如该文：https://sci-hub.se/10.1111/ijlh.13349

**Comment by chunibyocola** [comment, marked as answer]
Edge 自带的 PDF 阅读器使用 `embed` 标签，是不允许被注入脚本的，但你可以先右键另存为保存到本地后再从本地打开。
